tracpro/polls/tasks.py

```diff
@@ -130,3 +131,39 @@ def org_task(self, org):
         are no longer on the remote.
         """
         apps.get_model('polls', 'Poll').objects.sync(org)
+
+
+@task
+def sync_questions_categories(org, polls):
+    # Create new or update SELECTED Polls to match RapidPro data.
+    from tracpro.polls.models import Poll, Question
+
+    # Save the associated Questions for this poll here
+    # now that these polls have been activated for the Org
+    selected_poll_names, selected_polls = [], []
+    for poll in polls:
+        selected_poll_names.append(poll.name)
+        selected_polls.append(Poll.objects.get(id=poll.id))
```

Comment:
Since we're passing in the polls that we're interested in, these 4 lines aren't really necessary anymore.